src/MCPClient/tests/test_rights_from_csv.py

```diff
@@ -62,13 +62,13 @@ def test_rows_processed_and_database_content(self):
         assert models.RightsStatementRightsGrantedNote.objects.count() == 3
 
         # Test row 1
-        row_1_rights_statement = models.RightsStatement.objects.get(pk=1)
+        row_1_rights_statement = models.RightsStatement.objects.order_by('pk')[0]
```

Comment:
> It seems odd to see slice syntax without extra checks, e.g. for an empty data structure, but looking at the docs, I can't see that a queryset throws and error, so this seems okay.

`QuerySet` implements `object.__getitem__` so you should expect an `IndexError` to be raised if the index requested is outside the sequence. That's fine because we don't want to hide the error, I'm just being optimistic I guess but that should be ok as long as the 

> The rationale to replace .get(pk=1) seems to be that we only care about the first row, and its id might not always be 1?

Yeah. I think that these tests were probably written against SQLite and MySQL's autoincrement is behaving different, or viceversa. We should probably figure that out. I think that the new approach is safer in general. I also think that we're still setting the `settings.test` module when we run our tests (see pytest.ini) but it would be good to double check.

Reply:
See https://www.sqlite.org/autoinc.html.

Reply:
```
In [4]: len(UserProfile.objects.all())
Out[4]: 1

In [5]: UserProfile.objects.all()[1]
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-5-1678061bb5d2> in <module>()
----> 1 UserProfile.objects.all()[1]

/usr/local/lib/python2.7/site-packages/django/db/models/query.pyc in __getitem__(self, k)
    199         qs = self._clone()
    200         qs.query.set_limits(k, k + 1)
--> 201         return list(qs)[0]
    202
    203     def __and__(self, other):

IndexError: list index out of range
```